DigitalLearningSolutions.Data.Migrations/Properties/Resources.Designer.cs

```diff
@@ -501,22 +501,22 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to --DLSV2-95 Removes System Versioning to auditable tables (DOWN)
-        ///
-        ///
-        ///-- Remove versioning from FrameworkCompetencies table
-        ///ALTER TABLE FrameworkCompetencies SET (SYSTEM_VERSIONING = OFF);
-        ///DROP TABLE dbo.FrameworkCompetencies;
-        ///DROP TABLE dbo.FrameworkCompetenciesHistory;
-        ///GO
-        ///
-        ///-- Remove versioning from FrameworkCompetencyGroups table
-        ///ALTER TABLE FrameworkCompetencyGroups SET (SYSTEM_VERSIONING = OFF);
-        ///DROP TABLE dbo.FrameworkCompetencyGroups;
-        ///DROP TABLE dbo.FrameworkCompetencyGroupsHistory;
-        ///GO
-        ///
-        ///--  [rest of string was truncated]&quot;;.
+        ///   Looks up a localized string similar to --DLSV2-95 Removes System Versioning to auditable tables (DOWN)
+        ///
+        ///
+        ///-- Remove versioning from FrameworkCompetencies table
+        ///ALTER TABLE FrameworkCompetencies SET (SYSTEM_VERSIONING = OFF);
+        ///DROP TABLE dbo.FrameworkCompetencies;
+        ///DROP TABLE dbo.FrameworkCompetenciesHistory;
+        ///GO
+        ///
+        ///-- Remove versioning from FrameworkCompetencyGroups table
+        ///ALTER TABLE FrameworkCompetencyGroups SET (SYSTEM_VERSIONING = OFF);
+        ///DROP TABLE dbo.FrameworkCompetencyGroups;
+        ///DROP TABLE dbo.FrameworkCompetencyGroupsHistory;
+        ///GO
+        ///
+        ///-- Remove versioni [rest of string was truncated]&quot;;.

```

Comment:
Is this a typo?

Reply:
I guess there is a typo in that SQL script? this is auto generated stuff

Reply:
It's just truncated at that point in one of several lines in a script that look like this "-- Remove versioning from Competencies table"

Reply:
Weird, did you maybe have the window highlighted at some point? I've typed `git [whatever]` into a .cs file a dozen times. If it's autogenerated maybe we just leave it.

Reply:
That explains it, I couldn't work out what it was but of course it's 'versioni-ng'. I'll pass this then.

Reply:
I added a new SQL file into the resources, I guess it just decided to generate slightly differently this time